src/test/fuzz/fuzz.cpp

```diff
@@ -269,6 +293,21 @@ int main(int argc, char** argv)
     }
     const auto end_time{Now<SteadySeconds>()};
     std::cout << g_fuzz_target << ": succeeded against " << tested << " files in " << count_seconds(end_time - start_time) << "s." << std::endl;
+    if (tested && !ever_used_g_prng && seeded_g_prng_zero) {
+        std::cerr << "\n\n"
+                     "The fuzz target ("
+                  << g_fuzz_target
+                  << ") did NOT use the global random state over all tested files.\n\n"
+
+                     "However, it did use \n"
+                     "SeedRandomStateForTest(SeedRand::ZEROS) at the beginning \n"
+                     "of processing the fuzz input.\n\n"
+
+                     "This is not needed and may slow down fuzzing. Please \n"
+                     "remove it or add fuzz inputs that excersise it.\n\n"
+                  << std::endl;
+        std::exit(EXIT_FAILURE);
```

Comment:
Should this also be an `abort()`, or should the commit message be adjusted to say something else than "Abort"?